cli/tests/integrations/test_task.py

```diff
@@ -184,7 +184,7 @@ def test_log_two_tasks():
     assert stderr == b''
 
     lines = stdout.decode('utf-8').split('\n')
-    assert len(lines) == 23
+    assert len(lines) == 19
```

Comment:
This seems like a very unstable assumption, since we may add/remove lines from executor logging (or change `test-app`) in any given release. Can you instead look for some expected text, like "Subscribed executor" or "Starting task"?